lib/membrane_audio_mixer.ex

```diff
@@ -109,6 +109,8 @@ defmodule Membrane.AudioMixer do
         |> Map.from_struct()
         |> Map.put(:pads, %{})
         |> Map.put(:mixer_state, initialize_mixer_state(caps, options))
+        # last_timestamp represents latest mixed audio timestamp that was sent
+        |> Map.put(:last_timestamp, 0)
```

Comment:
I'm not fond of haivng comments like that in the code. What do you think about renaming this key to `last_ts_sent`?